configure.js

```diff
@@ -33,7 +33,13 @@ exports.init = function() {
         state.install = render(ctx, {})
         pluginParser.loadPlugins(function(data){
             // actions can only be bound to grid after full initialization
-            state.install.set("plugins", data).then(libplugin.bindDeleteButtons(state));
+            var temp_plugins;
+            temp_plugins = state.install.get("plugins");
+            if (temp_plugins == undefined) {
+                temp_plugins = [];
+            }
+            temp_plugins.push(data);
+            state.install.set("plugins", temp_plugins).then(libplugin.bindDeleteButtons(state));
```

Comment:
Is there a way to bind the delete button of a single plugin, the one passed as argument to data?